src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/BuildingUnit/AttachAddressToBuildingUnitLambdaRequest.cs

```diff
@@ -0,0 +1,61 @@
+namespace BuildingRegistry.Api.BackOffice.Handlers.Lambda.Requests.BuildingUnit
+{
+    using Abstractions;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using BuildingRegistry.Api.BackOffice.Abstractions.Building.Validators;
+    using BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Commands;
+    using Sqs.Requests.BuildingUnit;
+
+    public sealed record AttachAddressToBuildingUnitLambdaRequest :
+        BuildingUnitLambdaRequest,
+        IHasBackOfficeRequest<AttachAddressToBuildingUnitBackOfficeRequest>
+    {
+        public AttachAddressToBuildingUnitBackOfficeRequest Request { get; }
+
+        public int BuildingUnitPersistentLocalId { get; }
+
+        public AttachAddressToBuildingUnitLambdaRequest(
+            string messageGroupId,
+            AttachAddressToBuildingUnitSqsRequest sqsRequest)
+            : this(
+                messageGroupId,
+                sqsRequest.BuildingUnitPersistentLocalId,
+                sqsRequest.TicketId,
+                sqsRequest.IfMatchHeaderValue,
+                sqsRequest.ProvenanceData.ToProvenance(),
+                sqsRequest.Metadata,
+                sqsRequest.Request)
+        { }
+
+        public AttachAddressToBuildingUnitLambdaRequest(
+            string messageGroupId,
+            int buildingUnitPersistentLocalId,
+            Guid ticketId,
+            string? ifMatchHeaderValue,
+            Provenance provenance,
+            IDictionary<string, object?> metadata,
+            AttachAddressToBuildingUnitBackOfficeRequest request)
+            : base(messageGroupId, ticketId, ifMatchHeaderValue, provenance, metadata)
+        {
+            BuildingUnitPersistentLocalId = buildingUnitPersistentLocalId;
+            Request = request;
+        }
```

Comment:
Only 1 ctor should survive, one was for testing iirc. So move that to testing lib.